src/gentropy/datasource/ensembl/vep_parser.py

```diff
@@ -738,14 +738,14 @@ def process_vep_output(
                             transcript.distance.alias("distance"),
                             transcript.gene_id.alias("targetId"),
                             transcript.impact.alias("impact"),
-                            transcript.transcript_id.alias("transcriptId"),
                             transcript.lof.cast(t.StringType()).alias(
                                 "lofteePrediction"
                             ),
                             transcript.lof.cast(t.FloatType()).alias("siftPrediction"),
                             transcript.lof.cast(t.FloatType()).alias(
                                 "polyphenPrediction"
                             ),
+                            transcript.transcript_id.alias("transcriptId"),
```

Comment:
That seems to be the cause, why we can not produce `variant_index` with the latest dev version